src/main/java/core/basesyntax/FigureSupplier.java

```diff
@@ -0,0 +1,41 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class FigureSupplier {
+    private static final int Figure_Count = 5;
+    private static final Color DEFAULT_COLOR = Color.WHITE;
+    private static final int DEFAULT_RADIUS = 10;
+    private static final int Max_Value = 25;
+    private static final Random random = new Random();
+    private ColorSupplier supplier = new ColorSupplier();
+
+    public Figure getRandomFigure() {
+
+        switch (random.nextInt(Figure_Count)) {
+            case 0:
+                return new Circle(supplier.getRandomColor(), random.nextInt(Max_Value));
```

Comment:
let's create separate private methods for each figure, that will be returning a random figure, for example: 
case 0:
     return getRandomCircle();
case 1:
     return getRandomSquare();
...

It would be easier to read your code